src/pages/admin/VerifyStaff.jsx

```
@@ -1,93 +1,271 @@
-      verificationStatus: "verified",
+      verificationStatus: 'approved',
       updatedAt: serverTimestamp(),
     });
-    // No manual refresh required; the user's own session gets the change via onSnapshot in auth-context.
+    // Also reflect status on the request document for UI consistency (non-breaking)
+    try {
+      await updateDoc(doc(db, 'verification_requests', v.id), {
+        status: 'approved',
+        reviewedAt: serverTimestamp(),
+      });
+    } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)